src/Resources/Resources/help/Remove-AzDeployment.md

```diff
@@ -31,20 +53,41 @@ Remove-AzDeployment -InputObject <PSDeployment> [-AsJob] [-PassThru] [-ApiVersio
 ```
 
 ## DESCRIPTION
-The **Remove-AzDeployment** cmdlet removes an Azure deployment at subscription scope and any associated operations.
+The **Remove-AzDeployment** cmdlet removes an Azure deployment and its associated operations at subscription, resource group, management group, or tenant scope.
 
 ## EXAMPLES
 
-### Example 1: Remove a deployment with a given name
+### Example 1: Remove a deployment at subscription scope
 ```
-PS C:\>Remove-AzDeployment -Name "RolesDeployment"
+PS C:\>Remove-AzDeployment -ScopeType "Subscription" -Name "RolesDeployment"
```

Comment:
@Tiano2017 same comment about adding the corresponding output to all examples in this file (if this cmdlet does return any output)